frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -0,0 +1,233 @@
+/* eslint-disable no-unused-vars, no-undef, no-await-in-loop, no-console */
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '../../../../../integration-tests/protractor.conf';
+import * as vmView from '../../views/virtualMachine.view';
+import { nameInput, errorMessage } from '../../views/wizard.view';
+import {
+  resourceTitle,
+  isLoaded,
+  filterForName,
+  resourceRowsPresent,
+} from '../../../../../integration-tests/views/crud.view';
+import {
+  selectDropdownOption,
+  waitForStringNotInElement,
+  resolveTimeout,
+  tryNTimes,
+} from '../utils/utils';
+import { vmConfig } from '../utils/types';
+import {
+  PAGE_LOAD_TIMEOUT,
+  VM_BOOTUP_TIMEOUT,
+  VM_STOP_TIMEOUT,
+  VM_ACTIONS_TIMEOUT,
+  WIZARD_CREATE_VM_ERROR,
+  UNEXPECTED_ACTION_ERROR,
+  TABS,
+  WIZARD_TABLE_FIRST_ROW,
+  DASHES,
+} from '../utils/consts';
+import { listViewAction } from '../../views/vm.actions.view';
+import { KubevirtDetailView } from './kubevirtDetailView';
```

Comment:
It looks like lint doesn't like the order here, `KubevirtDetailView` should be below `rowForName`:
https://github.com/openshift/console/pull/2066/commits/2478b2fd2b561dde0428175a8d9ad79f0caa30f0